internal/web/discussions.go

```diff
@@ -103,6 +123,7 @@ func (s *server) handlePackageDiscussionPage(w http.ResponseWriter, r *http.Requ
 		"ShowDiscussionLink": true,
 		"PackageHref":        pkgHref,
 		"DiscussionHref":     fmt.Sprintf("%s/discussion", pkgHref),
+		"AutoUpdate":         d.AutoUpdate,
```

Comment:
The auto update value only depends on the `pkg`, which is already contained in the `packageDetailPageContext`. So just as in the package detail pages, it would be easier to use `clientutils.AutoUpdateString(pkg)` here, which also takes care if `pkg` is nil.

Reply:
ahh nice, missed this earlier. done.